### PR TITLE
chore(release): update changelog and version to v0.20.0

### PR DESCRIPTION
**Pull request by shumkov**
## Issue being fixed or feature implemented

Release Dashmate v0.20.0

## What was done?

- updated changelog
- bump version

## How Has This Been Tested?

None

## Breaking Changes

None

## Checklist:

- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have added or updated relevant unit/integration/functional/e2e tests
- [x] I have made corresponding changes to the documentation

**For repository code-owners and collaborators only**
- [x] I have assigned this pull request to a milestone

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,3 +1,43 @@
...
+
+### Features
+
+* wait nodes to be ready ([#369](https://github.com/dashevo/dashmate/issues/369))
+* add a temporary reset script for the local network ([#361](https://github.com/dashevo/dashmate/issues/380), [#371](https://github.com/dashevo/dashmate/issues/380), [#380](https://github.com/dashevo/dashmate/issues/380))
+* add an option to enable debug logs ([#349](https://github.com/dashevo/dashmate/issues/349))
+* enable debug logs option for setup local command ([#362](https://github.com/dashevo/dashmate/issues/362))
+* enable and request miner interval for setup local ([#360](https://github.com/dashevo/dashmate/pull/360))
+* configure tenderdash log level ([#364](https://github.com/dashevo/dashmate/pull/364))
+* migrate Drive state tree to black3 ([#402](https://github.com/dashevo/dashmate/issues/402))
```
Comment (strophy) [inline CHANGELOG.md:24]: `blake3`

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,3 +1,43 @@
...
+* update DPP to 0.20.0 ([#381](https://github.com/dashevo/dashmate/issues/381))
+* update to Tenderdash v0.5.0  ([#358](https://github.com/dashevo/dashmate/issues/358), [#395](https://github.com/dashevo/dashmate/issues/395), [#399](https://github.com/dashevo/dashmate/issues/399))
+* efficient service build cache ([#389](https://github.com/dashevo/dashmate/issues/389))
+
+
+### BREAKING CHANGES:
+
+* setup local command requests miner interval and debug logs
+* Tenderdash v0.4.0 is not supported anymore
+* building services from path required compose v2 installed and `DOCKER_COMPOSE_V2` env to set
```
Comment (strophy) [inline CHANGELOG.md:36]: env to be set

File: CHANGELOG.md [hunk 3]
```diff
@@ -1,3 +1,43 @@
...
+* platform services 0.19 and lower are not supporting
```
Comment (strophy) [inline CHANGELOG.md:37]: not supported

**Review by jawid-h — APPROVED** [review]
👍

**Review by strophy — APPROVED** [review]
LGTM

**Pull request merged**